models/login_source.go

```diff
@@ -604,6 +604,9 @@ var SMTPAuths = []string{SMTPPlain, SMTPLogin}
 
 // SMTPAuth performs an SMTP authentication.
 func SMTPAuth(a smtp.Auth, cfg *SMTPConfig) error {
+	if cfg.TLS {
+		return SMTPAuthTLS(a,cfg)
+	}
 	c, err := smtp.Dial(fmt.Sprintf("%s:%d", cfg.Host, cfg.Port))
```

Comment:
```suggestion
	c, err := smtp.Dial(fmt.Sprintf("%s:%d", cfg.Host, cfg.Port))
```
Only non TLS links are used here